### ISSUE TITLE
Compilation error on latest UWP & Csla-Core

### ISSUE THREAD

**Issue by ajj7060** [MarimerLLC/cslaforum#599]
**Describe the bug**

After updating my UWP projects to the latest UWP and targeting Windows 10 build 1803, and updating to the latest Csla, I now get compilation errors that Serializable and Browsable are both defined in Csla and System.Runtime.

CS0433	The type 'SerializableAttribute' exists in both 'Csla, Version=4.7.101.0, Culture=neutral, PublicKeyToken=93be5fdc093e4c30' and 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'	Library-UWP

**Version and Platform**
CSLA version: 4.7.100
OS: Windows
Platform: UWP (targeting and min of Windows 1803)

**Code that Fails**
Any use of the `SerializableAttribute` OR `BrowsableAttribute`

**Stack Trace or Exception Detail**
Compilation error

**Additional context**
None.

**Comment by rockfordlhotka** [comment]
Are you sure you are referencing `CSLA-Core-NS` and not `CSLA-Core`?

Or to be more correct, are you referencing _only_ 'CSLA-UWP' and nothing else? Because it'll automatically bring in the correct core package as a dependency, and it is _far_ simpler to just reference the one package rather than manually trying to keep both in sync (when the tooling will do it for you).

**Comment by ajj7060** [comment]
No, I'm referencing Csla-Core.  Just didn't the update via the nuget package manager.  I'll try switching to Csla-Core-NS.  Is Csla-UWP needed for an assembly which only contains the business objects?

**Comment by rockfordlhotka** [comment]
No, but as per the release notes for the version, you need to use CSLA-Core-NS for all run times except full .NET.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ajj7060** [comment]
Thanks Rocky; switching to Csla-Core-NS fixed the issues for me.
